src/java.base/share/classes/java/nio/MappedMemoryUtils.java

```diff
@@ -164,4 +164,17 @@ private static long alignDown(long address, int pageSize) {
         // pageSize must be a power of 2
         return address & ~(pageSize - 1);
     }
+
+    // Returns the platform defined page size, which could be different from the one Bits.pageSize
+    // is aware of. If no implementation is provided, return Bits.pageSize instead.
+    private static int pageSize() {
+        int ps = pageSize0();
+        if (ps > 0) {
+            return ps;
+        } else {
+            return Bits.pageSize();
```

Comment:
`Bits::pageSize` returns `UnsafeConstants::PAGE_SIZE` which is injected by the VM. Could it be that that value is somehow derived differently on AIX?

Reply:
I believe so. See [the answer below](https://github.com/openjdk/jdk/pull/14904#issuecomment-1640454917) with regards to Dynamic Variable Page Sizes.